### ISSUE TITLE
Predict the quality of wine using various ML classifiers

### ISSUE THREAD

**Issue by ShashankP19** [WebClub-NITK/Hacktoberfest-2k18#33]
### Description
Given dataset with different features of wine, predict its quality. The quality is an integral score between 0 and 10.
The dataset can be found [here](https://archive.ics.uci.edu/ml/datasets/wine+quality)
The given features are
1. fixed acidity
2. volatile acidity
3. citric acid
4. residual sugar
5. chlorides
6. free sulfur dioxide
7. total sulfur dioxide
8. density
9. pH
10. sulphates
11. alcohol 

### Details
- Technical Specifications: python, scikit-learn, pandas, numpy
- Type of issue: Multiple issues
- Time Limit: 3 days for each classifier implementation

### Issue requirements / progress
Train the model on train data.  Predict target values using test data. Find accuracy of the model comparing with actual test data targets
- [ ] Using K Neighbors Classifier
- [ ] Using Decision Tree Classifier
- [ ] Using Gaussian Naive Bayes
- [ ] Using Support Vector Classifier
- [ ] Using Gaussian Process Classifier
- [ ] Using Random Forest Classifier
- [ ] Using Multi-layer Perceptron (MLP) Classifier
- [ ] Using AdaBoost Classifier
- [ ] Using Quadratic Discriminant Analysis

### Resources
- To help you get started, the starter code `wine_starter_code.ipynb` is present in `/machine_learning/wine_quality`
- Jupyter notebook [installation](http://jupyter.org/install.html)
- scikit-learn  [KNeighborsClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html)
- scikit-learn  [DecisionTreeClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html)
- scikit-learn  [GaussianNB](http://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.GaussianNB.html)
- scikit-learn  [SupportVectorClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html)
- scikit-learn  [GaussianProcessClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.gaussian_process.GaussianProcessClassifier.html#sklearn.gaussian_process.GaussianProcessClassifier)
- scikit-learn  [RandomForestClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html#sklearn.ensemble.RandomForestClassifier)
- scikit-learn  [MLPClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html#sklearn.neural_network.MLPClassifier)
- scikit-learn  [AdaBoostClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.AdaBoostClassifier.html#sklearn.ensemble.AdaBoostClassifier)
- scikit-learn  [QuadraticDiscriminantAnalysis](http://scikit-learn.org/stable/modules/generated/sklearn.discriminant_analysis.QuadraticDiscriminantAnalysis.html#sklearn.discriminant_analysis.QuadraticDiscriminantAnalysis)
- pandas [tutorial](https://www.tutorialspoint.com/python_pandas/)
- numpy [tutorial](https://www.tutorialspoint.com/numpy)

### Directory Structure
Place your solution file in path as follows.  
- For K Neighbors Classifier `/machine_learning/wine_quality/knn/<your_solution_file>`
- For Decision Tree Classifier `/machine_learning/wine_quality/dtc/<your_solution_file>`
- For Gaussian Naive Bayes `/machine_learning/wine_quality/gnb/<your_solution_file>`
- For Support Vector Classifier `/machine_learning/wine_quality/svc/<your_solution_file>`
- For Using Gaussian Process Classifier `/machine_learning/wine_quality/gpc/<your_solution_file>`
- For Using Random Forest Classifier `/machine_learning/wine_quality/rfc/<your_solution_file>`
- For Using Multi-layer Perceptron (MLP) Classifier `/machine_learning/wine_quality/mlp/<your_solution_file>`
- For Using AdaBoost Classifier `/machine_learning/wine_quality/abc/<your_solution_file>`
- For Using Quadratic Discriminant Analysis `/machine_learning/wine_quality/qda/<your_solution_file>`

**Comment by cemysf** [comment]
I would like to work on this

**Comment by ShashankP19** [comment]
@cemysf  each model requires a new pull request. Choose one of the models that you would want to work on. What model do you choose?

**Comment by ThaysPrado** [comment]
I would like to work on this. I will use Decision Tree Classifier

**Comment by AMosa3d** [comment]
Hey, I would like to work on MLP

**Comment by bilalvur** [comment]
I would like to work on random forest.

**Comment by ShashankP19** [comment]
@bilalvur You are assigned to work on Random Forest Classifier. Go ahead.

**Comment by cemysf** [comment]
Sorry for the late reply, I would like to work on Support Vector Classifier

**Comment by ShashankP19** [comment]
@cemysf You are assigned to work on Support Vector Classifier. Go ahead.

**Comment by GajeshS** [comment]
Hi, I would like to work on Quadratic Discriminant Analysis

**Comment by ShashankP19** [comment]
@GajeshS You are assigned Quadratic Discriminant Analysis. Go ahead.
